app/views/admin/edition_images/edit.html.erb

```diff
@@ -11,21 +11,21 @@
 
       <%= render "govuk_publishing_components/components/textarea", {
         label: {
-          text: "Caption",
+          text: "Caption and credit",
           heading_size: "l",
         },
         name: "image[caption]",
         id: "image_caption",
         value: image.caption,
-        rows: 5,
-        hint: tag.p("Name a person in a photo or label a graph, infographic or diagram. The caption appears next to the image.", class: "govuk-!-margin-bottom-0 govuk-!-margin-top-0"),
+        rows: 2,
+        hint: raw("Name a person in a photo or add image credit. <a class='govuk-link' href='//www.gov.uk/guidance/content-design/images'>Read the image copyright standards</a>. The text appears next to the image."),
```

Comment:
Stylistically this differs from the rest of Whitehall where all links are prefixed with `https` - a minor thing and something I'll leave at your discretion whether or not to change because the behaviour is the same.

Reply:
Ah, good point about the consistency, have added the protocol 👍